app/src/main/java/org/mozilla/fenix/search/SearchFragmentStore.kt

```diff
@@ -117,11 +117,13 @@ data class SearchFragmentState(
 /**
  * Creates the initial state for the search fragment.
  */
+@Suppress("LongParameterList")
 fun createInitialSearchFragmentState(
     activity: HomeActivity,
     components: Components,
     tabId: String?,
     pastedText: String?,
+    selectedFromMenuEngine: SearchEngine?,
```

Comment:
```suggestion
    searchEngine: SearchEngine?,
```
Could we also reorder this to be below `searchAccessPoint` to align with the param ordering you did above in `SearchDialogFragment`